src/main/java/sggw/wzim/czasnawypad/db/entity/AttractionRating.java

```diff
@@ -0,0 +1,35 @@
+package sggw.wzim.czasnawypad.db.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "attraction_ratings")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class AttractionRating {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "attraction_id", nullable = false)
+    private Attraction attraction;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @Column(nullable = false)
+    private Byte rating;
```

Comment:
Byte?

Reply:
in db we have tinyint so I thought it will be better than Integer